homeassistant/components/nmap_tracker/const.py

```diff
@@ -12,6 +12,6 @@
 # Interval in minutes to exclude devices from a scan while they are home
 CONF_HOME_INTERVAL: Final = "home_interval"
 CONF_OPTIONS: Final = "scan_options"
-DEFAULT_OPTIONS: Final = "-F -T4 --min-rate 10 --host-timeout 5s"
+DEFAULT_OPTIONS: Final = "-n -sn --disable-arp-ping --host-timeout 5s"
```

Comment:
I don't think we can remove `--min-rate 10` as users reported it ran forever withtout it

Reply:
Interesting, my first guess would be that happened because in the previous configuration nmap also did a port scan, not just host up/down. If you have any reports handy, I'd be interested to check the specifics of what they were doing.

The only reason I removed that (for myself) is due to the trade-off between speed/accuracy and possibility of over-fitting, since the threshold is most likely network dependent:

> Specifying a minimum rate should be done with care. Scanning faster than a network can support may lead to a loss of accuracy. In some cases, using a faster rate can make a scan take longer than it would with a slower rate. This is because Nmap's adaptive retransmission algorithms will detect the network congestion caused by an excessive scanning rate and increase the number of retransmissions in order to improve accuracy. So even though packets are sent at a higher rate, more packets are sent overall.

I've tried the two versions on my network out of curiosity:

### Before

```bash
time nmap -F -T4 --min-rate 10 --host-timeout 5s 192.168.0.0/24

Nmap done: 256 IP addresses (18 hosts up) scanned in 7.57 seconds

real    0m7,582s
user    0m0,201s
sys     0m0,100s
```

### After

```bash
time nmap -n -sn --disable-arp-ping --host-timeout 5s 192.168.0.0/24

Nmap done: 256 IP addresses (18 hosts up) scanned in 3.28 seconds

real    0m3,290s
user    0m0,028s
sys     0m0,028s
```

Reply:
@emilburzo More information/history can be found here: <https://github.com/home-assistant/core/issues/52565>